### ISSUE TITLE
other people on my city are dead

### ISSUE THREAD

**Issue by conejo22** [finndev/PokeBuddy#1030]
Hey guys!
The last week, all gyms have a lot of pokemon with +2500 CP (obvius are boted accounts). But now, all gyms have pokemons with 500~1200 CP...
I think, the people with pokes +2500 has been banned :) xDD

**Comment by Arieth** [comment]
Naaa, ill got everypoke at 2500+ incluied dragonite 3200 and with lv32 and ill still survived, now i will touch hood haha

**Comment by SWBThorben24** [comment]
38, 20 pokemons 2000+, 5 pokemons 3000+ (4 x dragonite XD), also not banned.

**Comment by conejo22** [comment]
im from argentina, mendoza.. 
all people here is banned xD i dont see pkmns +3000 any more

**Comment by AlexCtasy** [comment]
@conejo22 I from Argentina too dude, and days ago all I saw a lot of people level 36+ with 3000+ CP but today I guess all those players are banned, I have no time to put on fake gps to check the city gyms. But if is like u said that have no ppl with 3000 cp, is a great thing!

**Comment by AlexCtasy** [comment]
@SWBThorben24 Wow dude, lv 38, is so obviusly that u botted like a crazy, take carefull maybe u get banned soon xxaoxoaoxa.

**Comment by Rab89** [comment]
@ OP or maybe the hype trains over

**Comment by AlexCtasy** [comment]
Yeah the game tends to get boring, I getting boring too, first that i play this game alone btw i in red team but i play alone, never outside, love fake gps location xD
